### ISSUE TITLE
WARNING: Couldn't write lextab module 'lextab'. [Errno 20] Not a directory: '/usr/local/lib/python3.11/dist-packages/autoremove_torrents-1.5.4-py3.11.egg/autoremovetorrents/lextab.py'

### ISSUE THREAD

**Issue by blixten85** [jerrymakesjelly/autoremove-torrents#175]
/usr/local/bin/autoremove-torrents --conf=/home/pi/autoremove-torrents/config.yml --log=/home/pi/autoremove-torrents/logs
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.main INFO: Auto Remove Torrents 1.5.4
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.main INFO: Loading configurations...
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.main INFO: Found 1 task(s) in the file.
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: Running task 'my_task'...
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: Logging in...
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: Login successfully. The client is qBittorrent v4.3.9.
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: WebUI API version: 2.8.2 (v2)
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: Status reported by the client:
        Download Speed: 0.00B/s Total: 0.00B
        Upload Speed: 0.00B/s   Total: 191.94MiB
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: Getting all the torrents...
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.task INFO: Found 1 torrent(s) in the client.
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.strategy INFO: Running strategy my_strategy...
WARNING: Couldn't write lextab module 'lextab'. [Errno 20] Not a directory: '/usr/local/lib/python3.11/dist-packages/autoremove_torrents-1.5.4-py3.11.egg/autoremovetorrents/lextab.py'
Fri, 02 Jun 2023 15:28:30 autoremovetorrents.strategy INFO: Total: 0 torrent(s). 0 torrent(s) can be removed.

No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm

Architecture:            aarch64
  CPU op-mode(s):        32-bit, 64-bit
  Byte Order:            Little Endian
CPU(s):                  4
  On-line CPU(s) list:   0-3
Vendor ID:               ARM
  Model name:            Cortex-A72
    Model:               3
    Thread(s) per core:  1
    Core(s) per cluster: 4
    Socket(s):           -
    Cluster(s):          1
    Stepping:            r0p3
    CPU(s) scaling MHz:  100%
    CPU max MHz:         1500.0000
    CPU min MHz:         600.0000
    BogoMIPS:            108.00
    Flags:               fp asimd evtstrm crc32 cpuid
Caches (sum of all):
  L1d:                   128 KiB (4 instances)
  L1i:                   192 KiB (4 instances)
  L2:                    1 MiB (1 instance)
Vulnerabilities:
  Itlb multihit:         Not affected
  L1tf:                  Not affected
  Mds:                   Not affected
  Meltdown:              Not affected
  Mmio stale data:       Not affected
  Retbleed:              Not affected
  Spec store bypass:     Vulnerable
  Spectre v1:            Mitigation; __user pointer sanitization
  Spectre v2:            Vulnerable
  Srbds:                 Not affected
  Tsx async abort:       Not affected

**Comment by luckiestone** [comment]
unzip egg file to a folder with same name

**Comment by amlopin** [comment]
I am getting the same error. 

`WARNING: Couldn't write lextab module 'lextab'. [Errno 20] Not a directory: '/usr/local/lib/python3.9/dist-packages/autoremove_torrents-1.5.4-py3.9.egg/autoremovetorrents/lextab.py'`

So as you recommended I tried to create a folder with the same name, but it fails

`sudo mkdir autoremove_torrents-1.5.4-py3.9.egg
mkdir: cannot create directory ‘autoremove_torrents-1.5.4-py3.9.egg’: File exists`

Should I move the .egg to be able to create a folder, then to unzip it inside??

Thanks

**Comment by luckiestone** [comment]
> I am getting the same error.
> 
> `WARNING: Couldn't write lextab module 'lextab'. [Errno 20] Not a directory: '/usr/local/lib/python3.9/dist-packages/autoremove_torrents-1.5.4-py3.9.egg/autoremovetorrents/lextab.py'`
> 
> So as you recommended I tried to create a folder with the same name, but it fails
> 
> `sudo mkdir autoremove_torrents-1.5.4-py3.9.egg mkdir: cannot create directory ‘autoremove_torrents-1.5.4-py3.9.egg’: File exists`
> 
> Should I move the .egg to be able to create a folder, then to unzip it inside??
> 
> Thanks

Yes, and you can also name the folder to a different name, unzip, delete egg file ans then rename the folder. Anyway you want
